build.gradle

```diff
@@ -62,14 +63,19 @@ jacocoTestReport {
 jacoco {
     toolVersion = "0.8.6"
 }
+
 jacocoTestCoverageVerification {
     violationRules {
         rule {
+            includes = ['com/myapp/*']
+            excludes = [
+                    'com/myapp/Controller/*',
+            ]
```

Comment:
bardzo dziwnie działa ten sposób weryfikacji - niby reaguje na dodanie/usunięcie tego controller...ale zakomentowałem połowę testów a on nadal przechodzi, poza tym w taki sposób raport nadal pokazuje co innego. Spróbuj podpiąć to w afterEvaluate, tak jak jest tutaj (dla tasków verification i report):
https://github.com/andrzeju/invoicing-system-andrzej/blob/master/build.gradle